docs/index.rst

```diff
@@ -39,17 +39,17 @@ library. The easiest way to install is with pip::
 
 Then, URLs are just an import away::
 
-  from hyperlink import URL
+  import hyperlink
 
-  url = URL.from_text(u'http://github.com/python-hyper/hyperlink?utm_source=readthedocs')
+  url = hyperlink.parse(u'http://github.com/python-hyper/hyperlink?utm_source=readthedocs')
```

Comment:
FWIW I'm not a fan of `parse`, which can return two types, and would recommend that one use `URL.from_text` or `DecodedURL.from_text` directly instead, so this change seems like a minus to me, but I might be alone there.

Reply:
Yeah I think `parse()` was added to shift recommended practice toward DecodedURLs, because they're safer. `parse()` returns a `DecodedURL` by default. I think it also looks pretty nice.